packages/common/utils/src/array/isSubset.ts

```diff
@@ -0,0 +1,5 @@
+/** @tossdocs-ignore */
+
+export function isSubset(a: unknown[] = [], b: unknown[] = []): boolean {
+  return a.every(element => b.includes(element));
```

Comment:
Oh, I missed this

`includes` methods does not distinguish object type correctly

edge case is below

```ts
const foo = [[1,2,3], [4,5,6]];
const bar = [[1,2,3], [4,5,6], [7,8,9]];

isSubset(foo, bar); // Expected true, but false


const foo2 = [{a: 1}, {b: 1}];
const bar2 = [{a: 1}, {b: 1}, {c: 1}];

isSubset(foo2, bar2); // Expected true, but false

const foo3 = [new Set('1'), new Set('2')];
const bar3 = [new Set('1'), new Set('2'), new Set('3')];

isSubset(foo3, bar3); // Expected true, but false
```

If purpose of this function is compare of whole unknown type, you should make isEqual methods for unknown type such as lodash.isEqual

So, in my opinion, It is more convenient not unknown but `string | number`.  for simply find subset array using `includes` method

Reply:
For simple comparison, `string | number` seems more suitable than `unknown`.

Thank you for the great idea 😃